### PR TITLE
Improve commit message guidelines

### PR DESCRIPTION
**Pull request by helderco**
Signed-off-by: Helder Correia

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -93,20 +93,27 @@ The Signed-off-by line must match the **author's real name**, otherwise the PR w
...
+  - For instance, when doing a `git blame` on a file to figure out why a change
   was introduced, it's pretty meaningless to see a _Fix linter_ commit message.
   "Add Feature X" is much more meaningful.
 - Use `git rebase -i main` to group commits together and rewrite their commit message
 - To add changes to the previous commit, use `git commit --amend -s`. This will
   change the last commit (amend) instead of creating a new commit.
 - Format: Use the imperative mood in the subject line: "If applied, this commit
   will _your subject line here_"
+- Add the following prefixes to your commit message to help trigger automated processes[^1]:
+  - `docs:` for documentation changes only (e.g., `docs: Fix typo in X`);
```
Comment (helderco) [inline CONTRIBUTING.md:110]: Not sure if these examples are too much. At first I only add this on in the line above.

Comment (aluzzardi) [reply CONTRIBUTING.md:110]: Perfect!

**Review by aluzzardi — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (gerhard) [reply CONTRIBUTING.md:110]: These are great additions, thank you @helderco 🙌🏻

**Review by gerhard — APPROVED** [review]
Great improvements! 🚀

**Pull request merged**